src/IIIFPresentation/API/Features/Manifest/ManifestWriteService.cs

```diff
@@ -279,7 +280,8 @@ await canvasPaintingResolver.GenerateCanvasPaintings(request.CustomerId, request
                 }
             ],
             CanvasPaintings = canvasPaintings,
-            SpaceId = spaceId
+            SpaceId = spaceId,
+            LastProcessed = canvasPaintings?.Any(cp => cp.AssetId != null) ?? false ? null : DateTime.UtcNow
```

Comment:
If modified and created are both set to the exact same timestamp should this use that too? (I can't recall the reason for c + m using exact same value but I'm assuming it's for ease of identifying that it happened at the same timE)